### PR TITLE
Fix start up animation bug, memory and minor fixes

### LINKED ISSUE #689: Logo display incorrectly on ipad mini5
https://raw.githubusercontent.com/0x5826/tmp/master/2020-05-26%2009.41.16.jpg

### PR DESCRIPTION
**Pull request by macsleven**
- Fixed startup animation bug display on iPad mini

- Remove watch only wallet option from startup page 

- Localized strings

- Redesign startup animation to consume less memory.

- Require user to cancel sync before creating/deleting wallet.

- Beep only once per block(each wallet sends a block attached notification).
requires https://github.com/raedahgroup/dcrlibwallet/pull/140
closes #689 
Screenshot 2020-05-27 at 17 40 00
Screenshot 2020-05-27 at 17 41 48

### CODE REVIEW

File: Decred Wallet/vi.lproj/Localizable.strings [hunk 1]
```diff
@@ -263,6 +263,7 @@
 "extendedPublicKey" = "Extended public key";
 "import_" = "Import";
 "keyIsInvalid" = "Key is invalid";
+"disconnectAddWallet" = "Disconnect/Cancel sync before adding a wallet";
```
Comment (C-ollins) [inline Decred Wallet/vi.lproj/Localizable.strings:266]:
```suggestion
"disconnectAddWallet" = "Disconnect/cancel sync before adding a wallet";
```

File: Decred Wallet/vi.lproj/Localizable.strings [hunk 2]
```diff
@@ -285,6 +286,7 @@
 "notifications" = "Notifications";
 "incomingTransactions" = "Incoming transactions";
 "removeWalletFromDevice" = "Remove wallet from device";
+"disconnectDeleteWallet"= "Disconnect/Cancel sync before deleting wallet";
```
Comment (C-ollins) [inline Decred Wallet/vi.lproj/Localizable.strings:289]:
```suggestion
"disconnectDeleteWallet"= "Disconnect/cancel sync before deleting wallet";
```

File: Decred Wallet/zh-Hans.lproj/Localizable.strings [hunk 3]
```diff
@@ -263,6 +263,7 @@
 "extendedPublicKey" = "Extended public key";
 "import_" = "Import";
 "keyIsInvalid" = "Key is invalid";
+"disconnectAddWallet" = "Disconnect/Cancel sync before adding a wallet";
```
Comment (C-ollins) [inline Decred Wallet/zh-Hans.lproj/Localizable.strings:266]:
```suggestion
"disconnectAddWallet" = "Disconnect/cancel sync before adding a wallet";
```

File: Decred Wallet/zh-Hans.lproj/Localizable.strings [hunk 4]
```diff
@@ -285,6 +286,7 @@
 "notifications" = "Notifications";
 "incomingTransactions" = "Incoming transactions";
 "removeWalletFromDevice" = "Remove wallet from device";
+"disconnectDeleteWallet"= "Disconnect/Cancel sync before deleting wallet";
```
Comment (C-ollins) [inline Decred Wallet/zh-Hans.lproj/Localizable.strings:289]:
```suggestion
"disconnectDeleteWallet"= "Disconnect/cancel sync before deleting wallet";
```

File: Decred Wallet/Wallet Utils/TransactionNotification.swift [hunk 5]
```diff
@@ -107,9 +108,12 @@ extension TransactionNotification: DcrlibwalletTxAndBlockNotificationListenerPro
     
     func onBlockAttached(_ walletID: Int, blockHeight: Int32) {
         // View Controllers requiring this update should call
-        // `try? AppDelegate.walletLoader.multiWallet.add(self, uniqueIdentifier: "\(self)")`
-        if Settings.beepNewBlocks && !WalletLoader.shared.multiWallet.isSyncing() {
-            self.playSound()
+        // `try? AppDelegate.walletLoader.multiWallet.add(self, uniqueIdentifier: "\(self)")
```
Comment (C-ollins) [inline Decred Wallet/Wallet Utils/TransactionNotification.swift:111]: Delete comment

Comment (macsleven) [reply Decred Wallet/Wallet Utils/TransactionNotification.swift:111]: the comment is an explanation on how to use the function

**Pull request merged**